.github/actions/install-dashboards/action.yml

```diff
@@ -39,15 +39,15 @@ runs:
 
     - id: osd-version
       run: |
-        echo "::set-output name=osd-version::$(cat package.json | jq '.opensearchDashboards.version' | cut -c 2-4)"
-        echo "::set-output name=osd-x-version::$(cat package.json | jq '.opensearchDashboards.version' | cut -c 2-3)"
+        echo "::set-output name=osd-version::$(jq -r '.opensearchDashboards.version' package.json)"
+        echo "::set-output name=osd-x-version::$(jq -r '.opensearchDashboards.version | split(".") | .[0]' package.json).x"
```

Comment:
Here's what's happening in the second command:

- `split(".")`: This is another jq filter that splits the version string into an array of strings using the period as a delimiter. This would turn "2.10.0" into ["2", "10", "0"].
- `.[0]`: This is a jq filter that selects the first element of the array. This would extract the "2" from ["2", "10", "0"].
- `.x`: This is concatenated onto the end of the major version number. So if the major version number is "2", it becomes "2.x". (This was originally at the next step of branch switching and I moved to here to make the entire flow more straight forward.)